src/Core/Shop/LogoUploader.php

```diff
@@ -143,6 +171,11 @@ public function update($fieldName, $logoPrefix, array $files = [])
         return false;
     }
 
+    public function isSvgMimeType(string $mimeType): bool
+    {
+        return (bool) in_array($mimeType, self::SVG_MIMETYPES);
```

Comment:
```suggestion
        return in_array($mimeType, self::SVG_MIMETYPES);
```

`in_array` returns already a boolean